<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #c3e2d0</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#444444</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/[CITYPNG.COM]HD White Angle Pencil Icon PNG - 1000x1000.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/icons8-home-50.png</normaloff>img/icons8-home-50.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/_4E5F9257-C121-425C-BCD8-EDF1C92652CE_-removebg-preview.png</normaloff>img/_4E5F9257-C121-425C-BCD8-EDF1C92652CE_-removebg-preview.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>1041</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#808080</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>301</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>50</pointsize>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #9fc5e8;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>𝓟𝓮𝓷𝓬𝓲𝓵𝓮</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>90</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>20</pointsize>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Serve what you need.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>751</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>50</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Selling school supplies site</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1021</width>
         <height>351</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>img/school-supply-bg.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="indent">
        <number>-2</number>
       </property>
      </widget>
      <zorder>label_7</zorder>
      <zorder>label</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="btnnutLogIn">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>100</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
background-color: #8bea60;
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>𝙻𝚘𝚐 𝚒𝚗</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="btnnutSignIn">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>80</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
background-color: #444444;
border-radius: 20px</string>
       </property>
       <property name="text">
        <string>𝚂𝚒𝚐𝚗 𝚞𝚙</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>𝖣𝗈𝗇𝗍 𝗁𝖺𝗏𝖾 𝖺𝗇 𝖺𝖼𝖼𝗈𝗎𝗇𝗍?</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
